pkg/updater/resultstore/resultstore_test.go

```diff
@@ -488,6 +488,119 @@ func TestResultStoreColumnReader(t *testing.T) {
 	}
 }
 
+func TestCellMessageIcon(t *testing.T) {
+	cases := []struct {
+		name        string
+		annotations []*configpb.TestGroup_TestAnnotation
+		properties  map[string][]string
+		tags        []string
+		message     string
+		icon        string
+	}{
+		{
+			name: "basically works",
+		},
+		{
+			name: "find annotation from property",
```

Comment:
I think we want one more test case covering the behavior when there's multiple test annotations specified.